src/components/schedule-page/table/schedule/schedule-parts/base-row.component.tsx

```diff
@@ -57,7 +57,7 @@ export function BaseRowComponent(options: BaseRowOptions): JSX.Element {
               ...dataItem,
             }}
             cellIndex={cellIndex}
-            key={`${dataItem.value}${cellIndex}_${uuid}}`}
+            key={`${dataItem.value}_${cellIndex}_${uuid}`}
```

Comment:
To poprawia problem
My wcześniej źle używaliśmy kluczy, ponieważ mogły być zdublikowane.
I wtedy react przy zmienianiu wartości w komórce nie wiedział którą wartość zmienić i podmieniał dwie komórki.
![image](https://user-images.githubusercontent.com/21079319/110839498-e6e54f80-82a3-11eb-8563-5f4336b6d183.png)


Żeby to powtórzyć, na developie na lutym:
1.
![image](https://user-images.githubusercontent.com/21079319/110839551-f82e5c00-82a3-11eb-8c0a-20758ddad8c7.png)


2.  Zamieniamy na D2. 
Dla tej konkretnej komórki, przy poprzedniej implementacji, klucz by wyniósł 
D21_{uuid}
Problem jest taki, że ten pracownik ma w 21 dniu lutego zmianę D.
Inaczej mówiąc, ma już komórkę która ma klucz ustawiony na 
D21_{uuid}
![image](https://user-images.githubusercontent.com/21079319/110839941-6ffc8680-82a4-11eb-83a1-4e5ef9124204.png)

3. Zmieniamy na dowolną zmianę 
![image](https://user-images.githubusercontent.com/21079319/110839958-74c13a80-82a4-11eb-8894-9ca818f06693.png)


4. Widzimy efekt
![image](https://user-images.githubusercontent.com/21079319/110840127-a63a0600-82a4-11eb-9bde-8d92ceccd5a0.png)